File: Quest/Assets/Src/Cards/Ally Cards/SirLancelot.cs
using System.Collections;
using System.Collections.Generic;
using UnityEngine;

public class SirLancelot : AllyCard
{

    public SirLancelot() : base("Sir Lancelot", 15, 1) //base 5
    {

    }

    public override int getBattlePoints(GameState gState) //override 20
    {
        if (gState.getCurrentStoryCard().getName().Equals("Defend the Queen's Honor"))
        {
            return battlePoints = 25;
        }
        return battlePoints = 15;
    }
    

}

File: eclipse/Quest/src/model/cards/allyCards/SirLancelot.java
package model.cards.allyCards;

import model.cards.AllyCard;
import model.game.GameState;

public class SirLancelot extends AllyCard {

	public SirLancelot() {
		super("Sir Lancelot", 15, 0); //base 5
	}

    public int getBattlePoints(GameState gState) //override 20
    {
        if (gState.getCurrentStoryCard().getName().equals("Defend the Queen's Honor"))
        {
            return battlePoints = 25;
        }
        return battlePoints = 15;
    }
}
